### PR TITLE
Update dplyr to version 0.7.8

### PR DESCRIPTION
**Pull request by philip-khor**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by philip-khor** [comment]
@conda-forge-admin, please rerender

**Comment by jdblischak** [comment]
@philip-khor Thanks for the PR!

dplyr 0.7.8 has the following updated minimum dependencies:

    - r-rcpp >=0.12.19
    - r-rlang >=0.3.0
    - r-tibble >=1.4.2

**Comment by jdblischak** [comment]
@philip-khor Overall this looks great! I am first checking if we need to do anything special with the build number due to the compiler migration (https://github.com/regro/cf-scripts/issues/333). Once I get confirmation that the build number should be returned to 0, I will merge.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,11 +11,11 @@ source:
   url:
     - {{ cran_mirror }}/src/contrib/dplyr_{{ version }}.tar.gz
     - {{ cran_mirror }}/src/contrib/Archive/dplyr/dplyr_{{ version }}.tar.gz
-  sha256: d489cc8b53854ec30737bb7d39b331b67ca35f4275ad19e97420d7a247808330
+  sha256: 37799ac5734d14b14ab9594ddec7f79e9a798907b87ff8b2d281a148a14de51a
 
 build:
   merge_build_host: True  # [win]
-  number: 1001
+  number: 0
```
Comment (jdblischak) [inline recipe/meta.yaml]: Please change the build number to 1000. This is a temporary measure during the compiler migration

https://github.com/regro/cf-scripts/issues/333#issuecomment-444883624

**Comment by jdblischak** [comment]
The CircleCI build failed with

```
conda.CondaMultiError: ('Connection broken: OSError("(104, \'ECONNRESET\')",)', OSError("(104, 'ECONNRESET')",))
```

I'm not sure if that is a temporary connection error or not, so I restarted the CircleCI build.

**Comment by jdblischak** [comment]
Great. The CircleCI builds worked after a restart. Now the only remaining problem is the [openblas error on Travis](https://travis-ci.org/conda-forge/r-dplyr-feedstock/jobs/464357045#L526):

```
export SRC_DIR=/Users/travis/miniconda3/conda-bld/r-dplyr_1544107050667/work
dyld: Library not loaded: @rpath/libopenblas.dylib
  Referenced from: /Users/travis/miniconda3/conda-bld/r-dplyr_1544107050667/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/R/bin/exec/R
  Reason: image not found
/Users/travis/miniconda3/conda-bld/r-dplyr_1544107050667/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/R/bin/INSTALL: line 34:  3666 Done                    echo 'tools:::.install_packages()'
      3667 Trace/BPT trap: 5       | R_DEFAULT_PACKAGES= LC_COLLATE=C "${R_HOME}/bin/R" $myArgs --slave --args ${args}
Traceback (most recent call last):
  File "/Users/travis/miniconda3/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/cli/main_build.py", line 456, in main
    execute(sys.argv[1:])
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/cli/main_build.py", line 447, in execute
    verify=args.verify, variants=args.variants)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/api.py", line 208, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/build.py", line 2311, in build_tree
    notest=notest,
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/build.py", line 1477, in build
    cwd=src_dir, stats=build_stats)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/utils.py", line 374, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/utils.py", line 354, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['/bin/bash', '-e', '/Users/travis/miniconda3/conda-bld/r-dplyr_1544107050667/work/conda_build.sh']' returned non-zero exit status 133.
The command "conda build ./recipe -m ./.ci_support/${CONFIG}.yaml --clobber-file ./.ci_support/clobber_${CONFIG}.yaml" exited with 1.
```

**Comment by jdblischak** [comment]
I have the same openblas error on travis for r-bfa (https://github.com/conda-forge/r-bfa-feedstock/pull/1).

File: recipe/meta.yaml [hunk 2]
```diff
@@ -35,31 +33,33 @@ requirements:
...
     - r-glue >=1.1.1
     - r-magrittr >=1.5
     - r-pkgconfig >=2.0.1
     - r-plogr >=0.1.10
-    - r-rlang >=0.2.0
-    - r-tibble >=1.3.1
+    - r-rlang >=0.3.0
+    - r-tibble >=1.4.2
     - r-tidyselect >=0.2.3
+    - openblas                   # [osx]
```
[the remark this answers is not in the document]
Comment (dbast) [reply recipe/meta.yaml]: Thanks for clarifying that topic. Will also have a look at the packages where the openblas dependency was already merged.

**Comment by dbast** [comment]
It builds, nice!

**Comment by jdblischak** [comment]
Thanks @philip-khor and @dbast!

**Pull request merged**